e2e_test/js-tests/test_viem_tx.mjs

```diff
@@ -302,8 +308,8 @@ describe("viem send tx", () => {
 		});
 		const receipt = await publicClient.waitForTransactionReceipt({ hash });
 		assert.equal(receipt.status, "success", "receipt status 'failure'");
-		assert.isAtMost(receipt.effectiveGasPrice, maxFeePerGas, "effective gas price is too high");
-		assert.isAbove(receipt.effectiveGasPrice, Number(maxFeePerGas) * 0.7, "effective gas price is too low");
+		assert.isAtMost(Number(receipt.effectiveGasPrice), Number(maxFeePerGas), "effective gas price is too high");
+		assert.isAbove(Number(receipt.effectiveGasPrice), Number(maxFeePerGas) * 0.7, "effective gas price is too low");
```

Comment:
The reasoning for this fix:
>Casts from BigInt to Number because chai's assert functions reject BigInt arguments for integers comparison
Doesn't make sense to me, because this was working on the local network and actually the version on celo10 just worked for me on alfajores.

Reply:
It was failing when the e2e tests were merged to the celo10 branch - see the ci failures [here](https://github.com/celo-org/op-geth/actions/runs/11438667582/job/31820738285)

Reply:
@Kourin1996 So if you run this test on a local node, (so not exporting `NETWORK=alfajores`) does the test fail for you?

Reply:
No, all tests in `test_viem_tx.mjs` are passed with even local network (after restarting shell).

Reply:
I think I must have been running your version locally, it's definitely failing for me with the number error now that I check again.